README.md

```diff
@@ -1,8 +1,200 @@
 # Auto-heal Service
 
-This project contains the _auto-heal_ service. It receives alert
-notitifications from the Prometheus alert manager and executes Ansible
-playbooks to resolve the root cause.
+This project contains the _auto-heal_ service. It receives alert notifications
+from the Prometheus alert manager and executes Ansible playbooks to resolve the
+root cause.
+
+## Configuration
+
+Most of the configuration of the auto-heal service is kept in a YAML
+configuration file.  The name of the configuration file is specified using the
+`-config-file` command line option. If this option isn't explicitly given then
+the service will try to load the `autoheal.yml` file from the current working
+directory.
+
+In addition to the configuration file the auto-heal service also uses command
+line options to configure the connection to the Kubernetes API and the log
+level. Use the `-h` option to get a complete list of these command line options.
+
+The `-kubeconfig` command line option is used to specify the location of the
+Kubernetes client configuration file. When running outside of a Kubernetes
+cluster the auto-heal service will use `$HOME/.kube/config` by default, the same
+used by the `kubectl` command. When running inside a Kubernetes cluster it will
+use the configuration that Kubernetes mounts automatically in the pod file
+system. So in most cases this command line option won't have to be explicitly
+included.
+
+The `-alsologtostderr` option is very convenient when running the auto-heal
+service, both in development and production environments.
```

Comment:
Might be helpful to always log to stdout (so we could get the logs with `oc logs`)

Reply:
I agree. We can do that in a follow-up patch.

Reply:
@moolitayer I opened issue #9 to track that.